python/pip_install/private/generate_whl_library_build_bazel.bzl

```diff
@@ -83,6 +87,16 @@ py_library(
     deps = {dependencies},
     tags = {tags},
 )
+
+alias(
+   name = "{py_library_public_label}",
+   actual = "{py_library_actual_label}",
+)
```

Comment:
The new alias should be public, but the `impl` targets should be IMHO private. So an explicit visibility setting would be good.